src/plat/win32/detect.c

```diff
@@ -249,15 +261,20 @@ void detect_gpu(void)
 */
 void detect_disk(void)
 {
-	FILE *disk_file;
-	char drive[MAX_STRLEN];
 	char *disk_unit;
 	long long totalBytes, freeBytes, usedBytes, disk_used, disk_total,
 		disk_percentage;
 
+#ifdef __MINGW32__
+	LPCTSTR drive = "C:\\";
```

Comment:
I think there's some way to detect the system root drive, something like a `%SYSTEMROOT%` environment variable, but assuming `C:\` should be safe.